.github/workflows/ci.yml

```
@@ -173,6 +173,23 @@ jobs:
       # Once cargo's `resolver = "2"` is stable (~ MSRV Rust 1.52), remove this.
       PYO3_CI: 1
 
+
+  docs:
+    name: docs
+    needs: [fmt]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+         toolchain: nightly
+         override: true
+      - name: "Build docs with docsrs flag"
+        run: cargo doc --no-deps --features=default,multiple-pymethods,extension-module,num-bigint,num-complex,hashbrown,serde,nightly
```

I think let's make this string match the others in the file, so that it's easy to correct later with find-and-replace when features change.

```suggestion
        run: cargo doc --no-deps --no-default-features --features "macros num-bigint num-complex hashbrown serde multiple-pymethods"
```

The features I took out (`extension-module`, `nightly`) don't affect visible APIs so no need to pass them here.

---

`nightly` exposes some conversions that play with the specialization feature which *are* part of the public api (if the nightly flag is on). Or is it the case that you just don't want to expose them as such?